File: src/3019.number-of-changing-keys.3312/solution.ts
function countKeyChanges (s: string): number {
    s = s.toLowerCase();
    let res = 0;
    for (let i = 1; i < s.length; i++) {
        if (s[i] !== s[i - 1]) {
            res++;
        }
    }
    return res;
}

File: src/3019.number-of-changing-keys.3312/solution.java
class Solution {
    public int countKeyChanges(String s) {
        char[] chars = s.toLowerCase().toCharArray();
        int result = 0;
        for(int i=1;i<chars.length;i++){
            if(chars[i] != chars[i-1]){
                result++;
            }
        }
        return result;
    }
}